test/functional/p2p_addr_relay.py

```diff
@@ -11,14 +11,15 @@
     NODE_NETWORK,
     NODE_WITNESS,
     msg_addr,
-    msg_getaddr
+    msg_getaddr,
+    msg_verack
```

Comment:
nit: Would be nice to always add a trailing comma to avoid ugly diffs and keeping the `git blame` depth lower in the future.

Reply:
noted